Validate Function

djayard opened the pull request:
A work requirement led me to write a validate function. We required that checked in code be pretty, and the validate function helps us enforce that policy.

I'm opening the PR since it might be useful to other people.

biasso commented:
Hi @djayard
I already saw your fork and I was integrating the validate function with other changes.
It works slightly different.
Try to look at the [development branch](https://github.com/tarunc/gulp-jsbeautifier/tree/dev) and, if you want, let me know what you think.
I have to test it and make some corrections to the documentation, but now I have not much time.

If you want to try it (not for production):
`npm install git://github.com/tarunc/gulp-jsbeautifier.git#dev --save`

djayard commented:
@biasso 
I changed this PR's target to the dev branch and added some new commits. Please let me know if you have any questions about the file changes.

djayard commented on README.md:
```diff
@@ -155,30 +155,25 @@ gulp.task('prettify', function() {
+  return gulp.src(['./*.css', './*.html', './*.js'])
     .pipe(prettify.validate())
     .pipe(prettify.reporter())
-    .on('error', function(error) {
```
This example doesn't work since nothing attempts to read from reporter. Without read attempts, 'error' and 'end' aren't emitted.

biasso commented:
Thank you! It seems all right.
I will publish tomorrow the new version.
